src/api/schemas.ts

```diff
@@ -282,6 +281,7 @@ export const ApiStatusResponse = Type.Object(
     status: Type.String(),
     block_height: Type.Optional(Type.Integer()),
     max_inscription_number: Type.Optional(Type.Integer()),
+    max_cursed_inscription_number: Type.Optional(Type.Integer()),
```

Comment:
why having this upper bound?

Reply:
This actually just reports the max cursed number we currently have in the DB when you visit the `/ordinals` root endpoint. It won't put a cap on incoming inscriptions.